Assets/Editor/CustomGui.cs

```diff
@@ -21,6 +21,14 @@ public class CustomGUI : EditorWindow
     public List<int> found; // A list of found trees.
     public List<int> treesToDelete; // A list of trees to delete
     public int layers; // The amount of layers in the Tree.
+    int GUIwidth;
+    int GUIheight;
+   
+    //variables that affect the lines thickness, colour, etc...
+    int shadowLine = 1;
```

Comment:
Comment why magic number is 1.

Reply:
the Drawbezier function needs a thickness of a line to draw. the number 1 was chosen because it looks nice